web-module/src/main/webapp/templates/celAjax/struct/Tiny4Config.vm

```diff
@@ -129,52 +129,78 @@ $!services.rteconfig.setRteConfigHint("tinymce4")
 #set($filebaseURL = "$!{filebaseURL}&src_doc=${filePickerSpace}.${filePickerDoc}&columns=10")
 #set($filebaseURL = "$!{filebaseURL}&root=${filePickerSpace}&hasUpload=$hasUpload")
 ##
+## lookup the rte_styles field in the XWiki and WebPreference
+#set($rteStyles = $services.rteconfig.getRTEConfigField('styles'))
+## get multilingual style names
+#set($mlRteStyleArray = [])
+#set($mlRteStyleName = '')
+#foreach($rteStyleElement in $rteStyles.split('[=;]'))
```

Comment:
according to [CELDEV-846](https://synjira.atlassian.net/browse/CELDEV-846) rte-styles are separated by `,`. please correct here or issue description
```suggestion
#foreach($rteStyleElement in $rteStyles.split('[=,]'))
```

Reply:
sorry, CELDEV-846 was wrong. I corrected it. It is the old tinymce3 configuration syntax thus we cannot change it.